parsers/cron_file_parser_test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+"""Tests for grr.parsers.cron_file_parser."""
+
+import os
+
+from grr.lib import flags
+from grr.lib import rdfvalue
+from grr.lib import test_lib
+from grr.parsers import cron_file_parser
+
+
+class TestCronTabParsing(test_lib.GRRBaseTest):
+  """Test parsing of cron files."""
+
+  def testCronTabParser(self):
+    """Ensure we can extract jobs from a crontab file."""
+    parser = cron_file_parser.CronTabParser()
+    client = "C.1000000000000000"
+    results = []
+
+    path = os.path.join(self.base_path, "crontab")
+    plist_file = open(path)
```

Comment:
the files are named cron_file_parser because ultimately windows sched tasks and *nix at jobs will live in here and they are "cron" things that aren't crontabs.